src/scss/theme.scss

```diff
@@ -1,5 +1,14 @@
 $theme-colors: (
   "primary": #5e60ce,
+  "text-muted": #adb2c1,
```

Comment:
Good use of themeing!

Reply:
Yeah, width of the input box will change as its parent's width changes